doc/whats_new.rst

```diff
@@ -38,6 +38,10 @@ API
 - :func:`~hnn_core.Dipole.write` and :func:`~hnn_core.Dipole.read_dipoles`
   now support hdf5 format for read/write Dipole object, by
   `Rajat Partani`_ in :gh:`648`
+  
+- Add ability to optimize parameters associated with evoked drives and plot 
+  convergence. User can constrain parameter ranges and specify solver,
+  by `Carolina Fernandez Pujol`_ in :gh:`652`
```

Comment:
you should leave this for the last when everyone has agreed the PR is ready for merge. This file is prone to rebase conflicts